keylime/tenant.py

```diff
@@ -582,6 +585,39 @@ def do_cvadd(self):
                 response.text,
             )
             sys.exit()
+        else:
+            numtries = 0
+            added = False
+
+            while not added:
+                reponse_json = self.do_cvstatus()
+                if reponse_json["code"] != 200:
+                    numtries += 1
+                    if numtries >= self.maxr:
+                        logger.error(
+                            'Agent ID %s still not appear as "added" on Verifier\'s database after %d tries',
```

Comment:
I agree with Thore about using "appear", it's too wishy-washy.

Reply:
fixed :-)